src/utilities/export.ts

```diff
@@ -0,0 +1,17 @@
+export function getTimestamp() {
```

Comment:
I suggest we use `moment` here. It's already a dependency. How about something like this:
`export function getTimestamp(format: string = "YYYY-MM-DD-HH-mm-SS")` (just check the momentjs docs to ensure I've got the right values there)

Reply:
@veggiesaurus This part is modified.